sdks/java/io/elasticsearch-tests/elasticsearch-tests-2/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -49,11 +49,11 @@
 
   private static final String ES_IP = "127.0.0.1";
   private static final int MAX_STARTUP_WAITING_TIME_MSEC = 5000;
-
+  private static int esHttpPort;
```

Comment:
will not be needed if you apply comments bellow

Reply:
yes, it can be done like the other tests even with the index mapping overloading => I'll do it myself on another PR